receipt_upload/receipt_upload/receipt_processing/photo.py

```
@@ -487,7 +460,7 @@ def process_photo(
             successful_clusters += 1
 
         except Exception as e:
-            print(f"Error processing cluster {cluster_id}: {e}")
+            logger.exception("Error processing cluster %d: %s", cluster_id, e)
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)